example/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
+  flutter_form_builder: ^9.4.1
```

Comment:
I don't know if add form builder is a right choice.
PROS:
- Can see a example with `flutter_form_builder` package, encouraging the use of them.
CONS:
- It may be implied that `form_builder_validators` only works for `flutter_form_builder`, which would discourage a group of users to use it.

Reply:
Ok, maybe we can add examples using form builder validator directly from the flutter_form_builder package.

Reply:
I like your approach Artur 😎. 
Remove it from here and in the future we will put it in the other package example.